SourceKitStressTester/Sources/StressTester/StressTester.swift

```diff
@@ -152,6 +152,9 @@ public class StressTester {
     var actions = generator
       .generate(for: tree)
       .filter { action in
+        if let offsetFilter = options.offsetFilter, let actionOffset = action.offset, offsetFilter != actionOffset {
```

Comment:
Can just do `if let offsetFilter = options.offsetFilter, offsetFilter != action.offset`

Reply:
Oh yes, I was a little worried about a `nil` `action.offset` matching a `nil` `offsetFilter` in `!=` but that’s not an issue because `offsetFilter` is not optional at this point. 👍